### ISSUE TITLE
Leere Standort-/Knotenbeschreibung verwerfen

### ISSUE THREAD

**Issue by gpunktschmitz** [asdil12/ff-map#171]
Hatte man mal etwas in Standort- oder Knotenbeschreibung eingetragen und möchte dieses Feld leer speichern, so wird das leere Feld nicht auf der Knoten-Seite aktualisiert, sondern der alte Wert bleibt bestehen.

**Comment by adrianschmutzler** [comment]
Zunächst: Das Verhalten ist so korrekt beschrieben.

Das ist allerdings Absicht: Ich kann anhand der Daten nicht unterscheiden, ob ein Wert absichtlich auf "leer" gesetzt wird, oder ob es sich um einen unabsichtlichen Reset des Routers handelt. In letzterem Fall ist aber sehr wertvoll, wenn dann nicht sofort alle Daten verloren gehen, sodass man den Router-Inhaber kontaktieren kann etc.

Um Werte zu löschen, ist es am praktikabelsten, entweder ein Leerzeichen ' ' oder ein anderes Füllzeichen (z.B. '-') in die jeweiligen Felder zu speichern.

**Comment by adrianschmutzler** [comment]
Ja, das ist es auch. Die strenge Lösung wäre alle Router mit Reset zu verwerfen.
Dann wüsste man bei einem Reset aber halt nichts mehr über den Router, um ggf. Maßnahmen zu ergreifen (außer der Besitzer kriegt es selbst mit). Klassisches Beispiel sind hier Loops, die durch einen Reset entstehen. Hier wäre es mit der "strengen" Lösung unmöglich, den Besitzer ausfindig zu machen.

Ein Kompromiss könnte so aussehen:
Bei den Koordinaten und der Kontaktadresse handelt es sich ja prinzipiell nicht um freiwillige Daten, sondern diese werden aufgrund "übergeordneten Interesses" als Pflichtangaben erfasst (steht auch so in den Datenschutzhinweisen auf dem Router). Hier besteht also - solange der Router betrieben wird - kein Recht auf Löschung. Diese beiden Daten könnte man also "wie bisher" erfassen, aber alles andere wird dann gelöscht/löschbar (Beschreibung, PositionComment).

**Comment by adrianschmutzler** [comment]
Zum Orphaned Router: Die werden irgendwann automatisch ganz gelöscht. Die Spanne ist nur sehr lange (ich glaube 180 oder 360 Tage).
